lib/ansible/module_utils/common/file.py

```diff
@@ -49,57 +49,60 @@ class FileLock:
     '''
     Currently FileLock is implemented via fcntl.flock on a lock file, however this
     behaviour may change in the future. Avoid mixing lock types fcntl.flock,
-    fcntl.lockf and module_utils.common.file.FileLock as it will certainly cause
+    fcntl.flock and module_utils.common.file.FileLock as it will certainly cause
```

Comment:
Does this need to be changed? It seems the documentation was originally warning against mixing `fcntl.flock` with `fcntl.lockf`, but now we have two references to the same method in the docs.

Reply:
Unintentional change, while trying lockf and then back to flock with search/replace. I'll fix it